DATA/production/qc-async/trditstpc.json

```
@@ -0,0 +1,75 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "type": "2",
+        "number": "42"
+      },
+      "monitoring": {
+        "url": "influxdb-unix:///tmp/telegraf.sock"
+      },
+      "consul": {
+        "url": "alio2-cr1-hv-aliecs:8500"
+      },
+      "conditionDB": {
+        "url": "alio2-cr1-hv-qcdb1.cern.ch:8083"
+      }
+    },
+    "tasks": {
+      "TRDDigits": {
+        "active": "true",
+        "taskName": "Digits",
+        "className": "o2::quality_control_modules::trd::DigitsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD"
+        },
+       "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        }
+      },
+      "TRDTracklets": {
+        "active": "true",
+        "taskName": "Tracklets",
+        "className": "o2::quality_control_modules::trd::TrackletsTask",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD"
+        }
+      },
+      "TRDPulseHeightTrackMatch": {
+        "active": "true",
+        "className": "o2::quality_control_modules::trd::PulseHeightTrackMatch",
+        "moduleName": "QcTRD",
```

but here you can put the proper task name. The convention is to avoid detector code in the name, as explained here: https://github.com/AliceO2Group/QualityControl/blob/master/doc/ModulesDevelopment.md#naming-convention
It's not yet applied to sync QC tasks, though.
```suggestion
      "TRDPulseHeightTrackMatch": {
        "active": "true",
        "taskName" : "PulseHeightTrackMatch",
        "className": "o2::quality_control_modules::trd::PulseHeightTrackMatch",
        "moduleName": "QcTRD",
```

---

done